src/main/java/net/cytonic/cytosis/config/CytosisSettings.java

```diff
@@ -25,6 +26,8 @@ public class CytosisSettings {
     public static boolean SERVER_PROXY_MODE = false;
     public static String SERVER_SECRET = "";
     public static int SERVER_PORT = 25565;
+    public static String SERVER_WORLD_NAME = "";
+    public static Pos SERVER_SPAWN_POS = new Pos(0,1,0);
```

Comment:
So this is a problem. We need to be able to set this in a config file, so we need a parser for it